frontend/src/pages/projects/screens/detail/storage/clusterTableColumns.ts

```diff
@@ -0,0 +1,33 @@
+import { SortableData } from '~/components/table';
+import { ClusterStorageNotebookSelection } from '~/pages/projects/types';
+
+export const storageColumns: SortableData<ClusterStorageNotebookSelection>[] = [
+  {
+    label: 'ID',
+    field: 'id',
+    sortable: false,
+    className: 'pf-v6-u-hidden',
+  },
+  {
+    label: 'Name',
+    field: 'name',
+    sortable: false,
+    width: 35,
+  },
+  {
+    label: 'Path format',
+    field: 'pathFormat',
+    sortable: false,
+    width: 25,
```

Comment:
There is a small change in the row width. when selecting between standard path format and custom. I guess it's because of the effect of the hidden id column.
Update widths could solve this problem
```suggestion
    width: 20,
```